File: 283-move-zeroes/283. Move Zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        vector<int> lst;
        for (int num: nums) {
            if (num != 0) {
                lst.push_back(num);
            }
        }
        for (int num: nums) {
            if (num == 0) {
                lst.push_back(num);
            }
        }
        for (int i = 0; i < nums.size(); i++) {
            nums[i] = lst[i];
        }
    }
};

File: 283-move-zeroes/283. Move Zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        lst = []
        for i in range(len(nums)):
            if nums[i] != 0:
                lst.append(nums[i])
        for i in range(nums.count(0)):
            lst.append(0)
        for i in range(len(nums)):
            nums[i] = lst[i]

       
            